Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:10-slim
```

Comment:
Is this node version correct?

Reply:
Hmm, `8.14.0-alpine` is better, can you change?

Reply:
@IgorHalfeld Docker Aline have some troubles with dns resolution when you try to run the container inside a Kubernetes Cluster so I think that slim image will be a better option.

https://github.com/gliderlabs/docker-alpine/issues/255